woocommerce/woocommerce-android
# Product selector SKU search: analytics tracking

Product selector SKU search: analytics tracking

### Description

This PR adds analytics event tracking for Product selector SKU search.
<img width="763" alt="Screenshot 2023-06-02 at 11 53 04" src="https://github.com/woocommerce/woocommerce-android/assets/4527432/f09daf17-1aeb-45b5-9867-6a73bea2e5d5">

Details here: pdfdoF-2Wl-p2

### Testing instructions

1. Open the product selector 
2. Use the search in "All products" and "SKU" modes
3. Verify `order_creation_product_selector_search_triggered` event is tracked with `search_filter` prop equal to `all` or `sku` accordingly.
### Images/gif

<img width="1296" alt="Screenshot 2023-06-02 at 11 50 30" src="https://github.com/woocommerce/woocommerce-android/assets/4527432/0ddc178a-15d2-4e64-9782-d32e71fc3902">
